<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User_Prefrence</class>
 <widget class="QDialog" name="User_Prefrence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#202A67;
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>21</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> color:#D6D3F6;
 border-radius: 10px;
background-color: #1D245B;
  border: 2px solid #F979AB ;

</string>
   </property>
   <property name="text">
    <string> User Prefrence  </string>
   </property>
  </widget>
  <widget class="QPushButton" name="Load_Button">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>200</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> color:#D6D3F6;
 border-radius: 10px;
background-color: #1D245B;
</string>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_User_Button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>410</y>
     <width>200</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> color:#D6D3F6;
 border-radius: 10px;
background-color: #1D245B;
</string>
   </property>
   <property name="text">
    <string>Add User</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Select_User">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>250</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color:#D6D3F6;
background-color: #1D245B;
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Select_Mode">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>250</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color:#D6D3F6;
background-color: #1D245B;
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Select_Theme">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>250</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color:#D6D3F6;
background-color: #1D245B;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Username">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>UserName:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Mode">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>97</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Mode:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Theme">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>90</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Theme:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/Downloads/car-png-logo-hd-transparent-png-50241-768x768.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
